src/shared/parameter/config_definitions/ssl_simulation_config.yaml

```diff
@@ -0,0 +1,27 @@
+- int:
+    name: team_channel
```

Comment:
curious why there are two yamls? the second seems to contain all the parameters in this file?

Reply:
[`ssl_simulation_main.cpp`](https://github.com/UBC-Thunderbots/SSL-Simulation-Protocol-Support/pull/11/files) shows how the configs are used, one is for command line args and the other is the config actually given to `SslSimulationProtocolManager`